README.md

```diff
@@ -1,2 +1,12 @@
 # Amundsen Marketing Site
-Built using NPM, Jekyll, and Tailwind.css via `npm start`. Hosted on github pages, deployed to `gh-pages` branch via `gulp deploy`.
+Built using NPM, Jekyll, and Tailwind.css via `npm start`.
+
+## Repo structure and branches
+We use GitHub pages to deploy amundsen.io. We use GitHub _organization_ pages (instead of _project_ pages). There are two ways to deploy GitHub pages:
```

Comment:
what is difference between organization page to a project page?

Reply:
Hey @feng-tao, as per [this link](https://docs.github.com/en/github/working-with-github-pages/about-github-pages#types-of-github-pages-sites), `Project sites are connected to a specific project hosted on GitHub, such as a JavaScript library or a recipe collection. User and organization sites are connected to a specific GitHub account.`

I choose an organization page because I saw this page more for the Amundsen organization and there seems to be a precedent for other projects too (like https://github.com/flyteorg/flyteorg.github.io).

Reply:
I will only merge once I get a +1 from you or Daniel:-)

Reply:
already +1 before

Reply:
Thanks!